pkgpanda/docs/for_packagers.md

```diff
@@ -1,109 +1,106 @@
 
-If there is one upstream package which you need
-protbuf-2.5.0-1
+If there is one upstream package which you need, this can be specified explicitly
 
-This works because
+* `protbuf-2.5.0-1`
 
 # Unsupported items
 
 ## Language-specific packages
 
-Currently packagepanda has no clue how, for instance,
+*Not currently implemented*
 
 ## Users
 
-Currently there isn't a good way of reliably adding users on a large number of machines which have had potentially non-identical config applied to them. Pkgpanda in a future release will utilize [Systemd Sysusers](http://www.freedesktop.org/software/systemd/man/systemd-sysusers.html) or something similar to provide the ability to manage users on a machine.
+*Not currently implemented*
 
+There isn't a good way of reliably adding users on a large number of machines which have had potentially non-identical
+config applied to them. Pkgpanda in a future release will utilize [Systemd Sysusers] or something similar to provide
+the ability to manage users on a machine.
+
+[Systemd Sysusers]: (http://www.freedesktop.org/software/systemd/man/systemd-sysusers.html)
 
 ## Fine-grained dependencies
 
-Not currently implemented. Eventually a package will be able to give the list of things it needs in it's operating environment (libraries, executables in path, files on filesystem, users). This is a much more robust way to state dependencies (At the cost of increased complexity, but the computation is still less than a second).
+*Not currently implemented.*
 
+Eventually a package will be able to give the list of things it needs in it's operating environment (libraries,
+executables in path, files on filesystem, users). This is a much more robust way to state dependencies (At the cost of
+increased complexity, but the computation is still less than a second).
 
 ## Depending on other packages
 
+Every package lives in isolation from other packages, and can be installed/uninstalled atomically.
 
+Packages are unpacked to a metadata file `pkginfo.json`, and Every package has a *Package Name* as well as a *Package
+Id*. Packages are unpacked to predictable locations so they may reference the absolute path to the file.
 
-Often ther
-
-
-
-Every package is ide
-
-Pkgpanda reliably ex
-
-Every package lives in isolation from other packages, and can be installed/uninstalled
-atomically. There is no notion of being able to edit things
+The `pkginfo.json` contains Packages depending on each other.
 
+**Well Known Directories**
 
+There are several well known directories in the tarball
 
-Packages are unpacked to
-
-a metadata file `pkginfo.json`, and
-
-
-Every package has a *Package Name* as well as a *Package Id*. Packages are unpacked
-to predictable locations so they may
-
-The `pkginfo.json` contains
-
-Packages depending on eachother. S
-
-
-There
-are several well known directories in the tarball
-
+```bash
 bin/
 lib/
 etc/
 dcos.target.wants
+```
+
+* environment variables
+* conflicts
+* Dependencies / requirements
```

Comment:
Remove these above bullet points and add as a TODO line item:
` * Document package environment variables, conflicts, and dependencies / requirements`